tester/python3/tester.py

```diff
@@ -83,9 +88,11 @@ def test(student_algorithm, test_cases_folder, name='alg', max_size=-1, only=[])
 			if expected_output[i] == alg_output[i]:
 				answers_ok.append(True)
 				print(u'\N{heavy check mark}', end='')
+				print(f'  (took {alg_running_time}s)', end='')
```

Comment:
could you add a space between the numbers and the `s`?

Reply:
added :)